src/index.js

```diff
@@ -145,6 +143,24 @@ export default function auth(config = {}) {
     // Register error handling middleware for redirecting to support
     // redirecting on authentication failure.
     app.use(middleware.failedLogin(authOptions));
+
+    // Setup route handler for default success redirect
+    if (authOptions.shouldSetupSuccessRoute) {
```

Comment:
What I'm saying is why are there default auth redirect HTML pages at all? I might be missing something but they don't seem particularly useful. If you do a browser redirect it'll pretty much always be to your app right? Couldn't we make it required and throw an error if it isn't set (or disabled)?